### ISSUE TITLE
support keras fit history in estimator's train_and_evaluate

### ISSUE THREAD

**Issue by karterotte** [alibaba/HybridBackend#76]
# User Story

I want to hold a record of the loss values and metric values during training, like keras History object:
https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/History
https://keras.io/guides/training_with_built_in_methods/
![image](https://user-images.githubusercontent.com/9114743/185031012-95c20126-4da3-45b7-a1cb-41e5281be848.png)

# Detailed requirements

I have to decide saving or not models depends on it's metrics(maybe lastest one).

# API Compatibility
hb.estimator.train_and_evaluate

# Willing to contribute

Yes

**Comment by 2sin18** [comment]
This feature request is completed in #79
